src/wily/__main__.py

```diff
@@ -406,12 +407,12 @@ def clean(ctx, yes):
 
     if not exists(config):
         logger.info(_("Wily cache does not exist, nothing to remove."))
-        exit(0)
+        sys.exit(0)
```

Comment:
Why call sys.exit instead of exit?

Reply:
my understanding is exit is for the interactive shell, while sys.exit is intended for use in programs.
https://docs.python.org/3/library/constants.html#exit vs https://docs.python.org/3/library/sys.html#sys.exit
as a result, pylint always recommends it: https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/consider-using-sys-exit.html